app/js/views/service.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'components/react';
 
-import CameraService from 'js/views/services/camera';
-import DefaultService from 'js/views/services/default';
+import BaseView from 'js/views/base-view';
+import CameraServiceView from 'js/views/services/camera';
+import DefaultServiceView from 'js/views/services/default';
 
-export default class Service extends React.Component {
+export default class Service extends BaseView {
```

Comment:
> Do we want to rename the top view classes too? i.e. ServiceView here.

I'd do it, but ideally in the consequent patches, like ```changing view - change it name``` :) And as you said we'll have time to think how we want to organize views.